draft-ietf-openpgp-pqc.md

```diff
@@ -565,14 +608,12 @@ It is given by the following algorithm, which computes the key encryption key `K
     //  domSep given in hexadecimal encoding := 4F 70 65 6E 50 47 50 43 6F 6D 70
```

Comment:
Not directly related to your PR but in earlier versions we distinguished "Input" and "Constants" for `multiKeyCombine`. I'd be in favor of listing `domSep` as constant again.

Reply:
I think it's a good idea to rename it to avoid confusion. I'm fine with it being `const`. I can't think of a good alternative with a more descriptive name right now.

You forgot to rename `domSep` in 9.2. and 9.2.1.

Reply:
I had an idea, see [55bea08](https://github.com/openpgp-pqc/draft-openpgp-pqc/pull/161/commits/55bea08b450e49dd8cb8a2c9c401fe9e63a37cec).

That is:
- Stick to `domSep` (don't rename it to `const`)
- We refer to `mlkemCT || mlkemPK || algId || domSep` as the OpenPGP-specific **context** information
- Refactored text in Section 9.2.1 to say that the inclusion of the ciphertexts binds the KEK to the specific session

Do we need the last paragraph in Section 9.2.1 that says something about ECC and SP800-56A? It feels like a leftover from old discussions. I removed it, if you think we need it, include it again.

Reply:
It's true that we discussed proposing to change the term "Domain" in the LAMPS draft. But when I think about it now, I fear that might delay the process. I would suggest to adopt to their terminology and find some other name for our string. Maybe "domsepString" or something similar?

Reply:
Why not leave our version as it is right now with this terminology. It is a only a proposal anyway, and as it is right now consistent in itself.

Reply:
I think it's ok this way. We could make it more clear that our additional inputs constitute the "Domain" field in the LAMPS draft. Would that work for you, Falko?

Reply:
It's OK for me, if there are complaints, we'll have to react.